cmd/bootkube/start.go

```diff
@@ -35,9 +37,14 @@ func init() {
 	cmdStart.Flags().StringVar(&startOpts.etcdServer, "etcd-server", "http://127.0.0.1:2379", "Single etcd node to use during bootkube bootstrap process.")
 	cmdStart.Flags().StringVar(&startOpts.assetDir, "asset-dir", "", "Path to the cluster asset directory. Expected layout genereted by the `bootkube render` command.")
 	cmdStart.Flags().BoolVar(&startOpts.selfHostedEtcd, "experimental-self-hosted-etcd", false, "Self hosted etcd mode. Includes starting the initial etcd member by bootkube.")
+	cmdStart.Flags().StringVar(&startOpts.flockPath, "apiserver-flock-path", "/var/lock/api-server.lock", "The flock path of the API server, used to prevent API server from port conflicts with the bootkube, not required when switched to static files.")
```

Comment:
I might just suggest hard-coding this to keep it relatively self-contained. The flock will soon just end up in the static manifest once https://github.com/kubernetes-incubator/bootkube/pull/425 lands. Which we should also coordinate this same change there (if it resolves the issue). /cc @diegs